source/extensions/filters/http/jwt_authn/data_store.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include "envoy/server/filter_config.h"
+#include "envoy/thread_local/thread_local.h"
+
+#include "common/common/logger.h"
+
+#include "extensions/filters/http/jwt_authn/extractor.h"
+#include "extensions/filters/http/jwt_authn/jwks_cache.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace JwtAuthn {
+
+/**
+ * The object to store config and caches. It has following objects:
+ * * config_: the config.
+ * * jwks_cache_ for Jwks cache.
```

Comment:
Nit: Be consistent in using `:` as a separator between these lines.

Reply:
Done